modules/meeting/app/components/meetings/side_panel/status_button_component.rb

```diff
@@ -47,8 +47,8 @@ def call
         OpPrimer::StatusButtonComponent.new(
           current_status: current_status,
           items: [open_status, in_progress_status, closed_status],
-          readonly: !edit_enabled?,
-          disabled: !edit_enabled?,
+          readonly: !edit_enabled? || @meeting.state == "draft",
+          disabled: !edit_enabled? || @meeting.state == "draft",
```

Comment:
```suggestion
          readonly: !edit_enabled? || @meeting.draft?,
          disabled: !edit_enabled? || @meeting.draft?,
```

We have the nice methods to check for enum states, we should use them